### ISSUE TITLE
New functionality broke the font fallback funtion

### ISSUE THREAD

**Issue by robvanderlee** [dompdf/dompdf#3364]
I've tested the font fallback feature for a while and when updating to the `2.0.3` or `2.0.4` release this somehow got broken. I've yet to dive into this with some more concrete pointers.

What I can tell is that when the feature got merged we ran on `dev-master#87bea32` which has a working result. The release after this commit (`2.0.3`) will not work, being the official release for this functionality in the first place :(

**Comment by bsweeney** [comment]
The font fallback functionality hasn't been officially released yet. [2.0.4 is just 2.0.3](https://github.com/dompdf/dompdf/compare/v2.0.3...v2.0.4) with a minor change to address a recursion vulnerability. The font fallback functionality will be released as part of Dompdf 3.0.0. I'm working through some updates in dependencies (primarily php-font-lib to fix issues with font parsing and subsetting) before I finalize the release.

In the meantime, you can access the fallback functionality by referencing dev-master in your composer configuration.

**Comment by 8ctopus** [comment]
@bsweeney Could you please tag the font fallback?

I'm using this is my package json:

```
    "require": {
        "dompdf/dompdf": "dev-master#05abdb3dbf51cb2263080b500a63ec483d5d4125",
    },
```

however the composer json that uses the package, nevertheless automatically updates to dev-master HEAD on `composer update` which is dangerous.

**Comment by 8ctopus** [comment]
Issue can be closed as 3.0.0 was released.

**Comment by bsweeney** [comment]
Thanks for reminding me.
